### DISCUSSION TITLE
Z_SCORING without mask

### DISCUSSION THREAD

**Discussion by xav69ier** [CBICA/CaPTk#1423] [Q&A] [answered]
Dear friends from CBICA 
I am using the command line to normalizing my scans (multimodal, multicentre, i.e. different patients scanned each one at different hospitals ) - https://cbica.github.io/CaPTk/preprocessing_zScoreNorm.html - however, I do not provide in the command line a mask to keep the skull in. HOwever I understand that the normalization will consider also the pixels outside the brain and the skull, and I wonder how bad or how distorting this might be for the correct normalization of my data, that I use to train a CNN for regression over my ground truth, another co-registered image, of course also without mask - for any comments I am most thankful: Xavier

**Comment by sarthakpati** [comment, marked as answer]
Hi @xav69ier,

This will cause issues during training of a network, because now the background (i.e., anything with `0`) will be scaled, as well. Regardless of whether you want to use a CNN or any other ML approach, it is advisable to pass the mask to ensure the background voxels are not scaled.

Cheers,
Sarthak

**Reply by xav69ier** [reply]
I agree fully - many thanks Sarthak x

**Reply by sarthakpati** [reply]
You're welcome! Please feel free to reach out if you have any questions! And thank you for your interest in CaPTk! 😄
